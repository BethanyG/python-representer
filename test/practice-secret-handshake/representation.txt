Module([FunctionDef('placeholder_0', arguments([], [arg('placeholder_1', lineno=1, col_offset=13, end_lineno=1, end_col_offset=23)], kwonlyargs=[], kw_defaults=[], defaults=[]), [Pass(lineno=2, col_offset=4, end_lineno=2, end_col_offset=8)], [], lineno=1, col_offset=0, end_lineno=2, end_col_offset=8)], [])