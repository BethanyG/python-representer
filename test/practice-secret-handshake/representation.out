## BEGIN NORMALIZED CODE ##
def placeholder_0(placeholder_1):
    pass

## END NORMALIZED CODE ##

## BEGIN TREE BEFORE ##
Module(
  [
    FunctionDef(
      'commands',
      arguments(
        [],
        [
          arg(
            'binary_str',
            lineno=1,
            col_offset=13,
            end_lineno=1,
            end_col_offset=23)],
        kwonlyargs=[],
        kw_defaults=[],
        defaults=[]),
      [
        Pass(
          lineno=2,
          col_offset=4,
          end_lineno=2,
          end_col_offset=8)],
      [],
      lineno=1,
      col_offset=0,
      end_lineno=2,
      end_col_offset=8)],
  [])
## END TREE BEFORE ##

## BEGIN NORMALIZED TREE ##
Module(
  [
    FunctionDef(
      'placeholder_0',
      arguments(
        [],
        [
          arg(
            'placeholder_1',
            lineno=1,
            col_offset=13,
            end_lineno=1,
            end_col_offset=23)],
        kwonlyargs=[],
        kw_defaults=[],
        defaults=[]),
      [
        Pass(
          lineno=2,
          col_offset=4,
          end_lineno=2,
          end_col_offset=8)],
      [],
      lineno=1,
      col_offset=0,
      end_lineno=2,
      end_col_offset=8)],
  [])
## END NORMALIZED TREE ##